custom_components/zwave_mqtt/climate.py

```diff
@@ -80,19 +54,9 @@
     "cool econ": ("setpoint_eco_cooling",),
     "away": ("setpoint_away_heating", "setpoint_away_cooling"),
     "full power": ("setpoint_full_power",),
-    # aliases found in xml configs
-    "comfort": ("setpoint_heating",),
-    "heat mode": ("setpoint_heating",),
-    "heat (default)": ("setpoint_heating",),
-    "dry floor": ("setpoint_dry_air",),
-    "heat eco": ("setpoint_eco_heating",),
-    "energy saving": ("setpoint_eco_heating",),
-    "energy heat": ("setpoint_eco_heating",),
-    "vacation": ("setpoint_away_heating", "setpoint_away_cooling"),
-    # for tests
-    "heat_cool": ("setpoint_heating", "setpoint_cooling"),
 }
 
+# strings, OZW and/or qt-ozw does not send numeric values
```

Comment:
This is not ideal... I can confirm, my tstat only shows string values. I've submitted a bug upstream, maybe it can be addressed.

https://github.com/OpenZWave/qt-openzwave/issues/65

Reply:
Thanks @kpine for the issue report to OZW.
For now we'll go with the string values